File: dal/src/main/java/com/bykea/pk/partner/dal/source/remote/response/GetWithdrawalPaymentMethods.java
package com.bykea.pk.partner.dal.source.remote.response;

import com.bykea.pk.partner.dal.source.remote.response.data.WithdrawPaymentMethod;

import java.util.List;

public class GetWithdrawalPaymentMethods extends BaseResponse {

    private List<WithdrawPaymentMethod> data;

    public List<WithdrawPaymentMethod> getData() {
        return data;
    }

    public void setData(List<WithdrawPaymentMethod> data) {
        this.data = data;
    }
}

File: dal/src/main/java/com/bykea/pk/partner/dal/source/remote/response/GetWithdrawalPaymentMethods.kt
package com.bykea.pk.partner.dal.source.remote.response

import com.bykea.pk.partner.dal.source.remote.data.WithdrawPaymentMethod
import com.google.gson.annotations.SerializedName

data class GetWithdrawalPaymentMethods(
        @SerializedName("data") val data: List<WithdrawPaymentMethod>?
) : BaseResponse() {
}